spec/results-model-spec.js

```diff
@@ -33,6 +35,10 @@ describe("ResultsModel", () => {
       expect(resultsModel.getPathCount()).toBe(1);
       expect(resultsModel.getMatchCount()).toBe(6);
       expect(resultsModel.getPaths()).toEqual([editor.getPath()]);
+      if (parseFloat(atom.getVersion()) >= 1.17) {
```

Comment:
Considering that this check is going to be short-term and only for specs I think it's safe to keep the float interpretation.